target/scripts/wrapper/getmail-wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+while true; do
+    for file in /etc/getmailrc.d/getmailrc*; do
+        #/usr/bin/getmail --getmaildir /etc/getmailrc.d --rcfile getmailrc.file
+        if ! pgrep -f "${file}"$; then
+          /usr/bin/getmail --getmaildir /var/lib/getmail --rcfile "${file}"
+        fi
+    done
+    sleep 300
+done
```

Comment:
`sleep 300` should be using `GETMAIL_POLL` ENV?

This file does not appear to be referenced anywhere in the PR, and supposedly replaced by the Cron job, is this file meant to be part of the PR?

As per the cronjob review comment, this could be repurposed to be part of a single cronjob, instead of one per user.

Reply:
yah you are correct, forgot to remove it.

Reply:
Repurposed as getmail-cron script.